src/Project.ts

```diff
@@ -280,6 +283,45 @@ export class Project {
 		}
 	}
 
+	private validateRbxTypes() {
+		const pkgLockJsonPath = path.join(this.projectPath, "package-lock.json");
+		if (fs.pathExistsSync(pkgLockJsonPath)) {
+			const pkgLock = JSON.parse(fs.readFileSync(pkgLockJsonPath).toString());
+			if (pkgLock !== undefined) {
+				const dependencies = pkgLock.dependencies;
+				if (dependencies !== undefined) {
+					const rbxTypes = dependencies["rbx-types"];
+					if (rbxTypes !== undefined) {
+						const rbxTypesVersion = rbxTypes.version;
+						if (typeof rbxTypesVersion === "string") {
+							const regexMatch = rbxTypesVersion.match(/\d+$/g);
+							if (regexMatch !== null) {
+								const patchNumber = Number(regexMatch[0]);
+								if (!isNaN(patchNumber)) {
+									if (patchNumber >= MINIMUM_RBX_TYPES_VERSION) {
+										return;
+									} else {
+										throw new ProjectError(
+											`rbx-types is out of date!\n` +
+												yellow(`Installed version: ${rbxTypesVersion}\n`) +
+												yellow(`Minimum required version: 1.0.${MINIMUM_RBX_TYPES_VERSION}\n`) +
```

Comment:
I don't necessarily like that it only allows versions which begin with `1.0.`. Are we going to have to "fix" this in the future due to an update?

Reply:
![image](https://user-images.githubusercontent.com/15217173/58516337-3f019f00-816d-11e9-816b-fc0b35ca0e30.png)